data/variable/database/templates/users.mysql.sql

```diff
@@ -2,7 +2,7 @@ CREATE TABLE users (
 
     id       INTEGER UNSIGNED NOT NULL PRIMARY KEY AUTO_INCREMENT,
     username VARCHAR(50),
-    digesta1 VARCHAR(32),
+    digesta1 VARCHAR(60),
```

Comment:
@Hywan why is it 60 actually? this database table was specific for digest, which only does MD5 ... are we simply re-using the same key names for non-digest?

Perhaps it's better to rename `digesta1` to `password` or `passwordhash` to reduce confusion?